src/services/booking.tsx

```diff
@@ -0,0 +1,22 @@
+const URL = "https://tech-test.joovence.dev/api";
```

Comment:
Since this is common to the API, maybe its a good idea to extract it as a constant and use for all endpoints
```suggestion
const API_BASE_URL = "https://tech-test.joovence.dev/api";
```